yt_dlp/extractor/zee5.py

```diff
@@ -176,7 +179,7 @@ class Zee5SeriesIE(InfoExtractor):
                      (?:
                         zee5:series:|
                         https?://(?:www\.)?zee5\.com/(?:[^#?]+/)?
-                        (?:tv-shows|web-series|kids|zee5originals)(?:/[^#/?]+){2}/
```

Comment:
why not just `[^#/?]+`?

Reply:
yup, old one seems fine.

Reply:
now there is a conflict of "kids-movies" matching zee5seriesIE regex